app/uk/gov/hmrc/selfservicetimetopay/controllers/CalculatorController.scala

```diff
@@ -80,12 +80,16 @@ class CalculatorController(calculatorConnector: CalculatorConnector) extends Tim
   }
 
   def submitAmountsDue: Action[AnyContent] = Action.async { implicit request =>
+    var loggedIn = false
```

Comment:
I was doing this because the result of the first map means the second map no longer has sessioncache in scope (that seemed to be the point of the first map/case block, to return a seq whether or not there was anything in cache). It's not an expert bit of work though so happy to see this cleaned up. mapping straight into case without "match" is a new scala shorthand trick.